planner_server:
  ros__parameters:
    service_name: "planner"
    config_file_name: "franka.yml"
    max_attempts: 3
    interpolation_dt: 0.01

    # ik
    rotation_threshold: 0.05
    position_threshold: 0.005
    num_seeds: 20
